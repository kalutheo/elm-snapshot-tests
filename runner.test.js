const chalk = require('chalk');
const Elm = require("./.elm-snapshots/PrivateMain.js");
const log = console.log;

describe(snapshots.test, () => {
  snapshots.actions.forEach((snapshot) => {
    test(snapshot.title, () => {
      expect(snapshot).toMatchSnapshot();
    })
  })
});
