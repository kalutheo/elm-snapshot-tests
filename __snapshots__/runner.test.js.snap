// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A Small person BMI Select a Gender 1`] = `
Object {
  "msg": "SelectGender Female",
  "newModel": "{ height = \\"180\\", weight = \\"80\\", gender = Female }",
  "title": "Select a Gender",
}
`;

exports[`A Small person BMI Select a Height 1`] = `
Object {
  "msg": "SelectHeight \\"150\\"",
  "newModel": "{ height = \\"150\\", weight = \\"80\\", gender = Female }",
  "title": "Select a Height",
}
`;

exports[`A Small person BMI Select a Weight 1`] = `
Object {
  "msg": "SelectWeight \\"55\\"",
  "newModel": "{ height = \\"180\\", weight = \\"55\\", gender = Female }",
  "title": "Select a Weight",
}
`;
